.github/workflows/test-many-configs.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: auto-run-manyConfigs-script
```

Comment:
I think this should change to "auto-run-test-many-configs-script" to match the name.